database/migrations/2024_03_23_074510_create_product_details_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('product_details', function (Blueprint $table) {
+            $table->id();
+            $table->text('size');
```

Comment:
use double for size and weight

Reply:
We want to include the units of size such as L:30cm W:15cm H:10cm.
For the weight the same reasons(ex.300g, 20kg). Seller can edit the size and weight in any style.